Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -47,14 +52,19 @@ void Start()
             gestureRecognizer.StartCapturingGestures();
         }
 
-        private void GestureRecognizer_TappedEvent(InteractionSourceKind source, int tapCount, Ray headRay)
+        private void SendMessageToFocusedObject()
```

Comment:
Maybe something like 'OnTap' would be a more clear name here? My real qualm is that this doesn't just send any message to the focused object it sends 'OnSelect' to the focused object. I think 'OnTap' makes it clear that the method should be called whenever the GestureManager recognizes a tap.